keyring.go

```diff
@@ -12,22 +12,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build !darwin
+//go:build linux
+// +build linux
```

Comment:
My fault sorry, I've just restricted `keyring.go` to Windows and Linux, adding an empty keyring interface for other platforms.

Reply:
No worries. This isn't an easy one to test because GitHub Actions is only set up for Linux, Mac and Windows.

I noticed that the github.com/zalando/go-keyring package has a "fallback" provider, so if we made this build constraint `!darwin`, we don't need the `no_keyring.go` file? The call to [src.getCredentials()
](https://github.com/samuong/alpaca/pull/136/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261L66) will result in an [ErrUnsupportedPlatform](https://github.com/zalando/go-keyring/blob/ead676fd21b75d0f7665884e8a299f7ec8dfc6d7/keyring_fallback.go#L18) error, and we'll fall back to no keyring.